pkg/apis/build/v1alpha2/build_conversion.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha2
+
+import (
+	"context"
+
+	"knative.dev/pkg/apis"
+
+	"github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
+)
+
+func (b *Build) ConvertTo(_ context.Context, to apis.Convertible) error {
+	toBuild := to.(*v1alpha1.Build)
```

Comment:
i think in the future we should look at doing something like this with the type switch https://github.com/tektoncd/pipeline/blob/67b318ba62d9dfee4e9ec639571754f42d9702d7/pkg/apis/pipeline/v1alpha1/cluster_task_conversion.go#L31-L50

Reply:
yea we should probably have this, updated